etsin_finder/resources/common_resources.py

```diff
@@ -19,21 +19,22 @@
 from etsin_finder.utils.qvain_utils import check_dataset_edit_permission
 
 from etsin_finder.services.common_service import (
-    get_directory_for_project,
+    get_directory_files_for_project,
     get_dataset_projects,
-    get_directory,
+    get_directory_files,
     get_dataset_user_metadata,
     update_dataset_user_metadata,
 )
 
 from etsin_finder.utils.log_utils import log_request
 
+TOTAL_ITEM_LIMIT = 5 * 1024 ** 4
```

Comment:
I think this is here accidentally, `TOTAL_ITEM_LIMIT` is supposed to be removed.